### PR TITLE
[service.subtitles.subztv.gr] 1.0.0

### PR DESCRIPTION
**Pull request by lambda81**
subtitle service for
http://subztv.gr/

### CODE REVIEW

File: service.subtitles.subztv.gr/resources/lib/subztvgr.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+            if len(files) == 0: return
+
+            control.execute('Extract("%s","%s")' % (f, path))
+
+            for i in range(0, 10):
+                try:
+                    dirs, files = control.listDir(path)
+                    if len(files) > 1: break
+                    control.sleep(1000)
```
Comment (learningit) [inline service.subtitles.subztv.gr/resources/lib/subztvgr.py]: Would be desirable to check for an abort here.

Comment (lambda81) [inline service.subtitles.subztv.gr/resources/lib/subztvgr.py]: I will add it in the next update of the script.

File: service.subtitles.subztv.gr/resources/lib/subztvgr.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    def download(self, path, url):
+        try:
+            result = client.request(url)
+
+            f = os.path.splitext(urlparse.urlparse(url).path)[1][1:]
+            f = os.path.join(path, 'file.%s' % f)
+
+            with open(f, 'wb') as subFile:
+                subFile.write(result)
+            subFile.close()
```
Comment (phil65) [inline service.subtitles.subztv.gr/resources/lib/subztvgr.py]: you dont need to close manually when using "with" statement.

Comment (lambda81) [inline service.subtitles.subztv.gr/resources/lib/subztvgr.py]: thanks, I forgot to remove it when I changed the code and used "with" instead
I have sent PRs with it removed

https://github.com/xbmc/repo-scripts/pull/256
https://github.com/xbmc/repo-scripts/pull/257

File: service.subtitles.subztv.gr/resources/lib/subztvgr.py (hunk 1 above)
Comment (lambda81) [inline service.subtitles.subztv.gr/resources/lib/subztvgr.py]: I have sent PRs with it
https://github.com/xbmc/repo-scripts/pull/256
https://github.com/xbmc/repo-scripts/pull/257

**Pull request merged**